src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -87,7 +81,7 @@ protected String getApiRoute() {
     public URL getPatchUrl() {
         return GitHub.parseURL(patch_url);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)